src/utils/map-controller.d.ts

```diff
@@ -5,18 +5,17 @@ export type MjolnirEvent = {
   type: string,
   center: {x: number, y: number},
   offsetCenter: {x: number, y: number},
-  deltaX: number,
-  deltaY: number,
-  delta: number,
-  scale: number,
-  rotation: number,
-  pointerType: string,
-  metaKey: boolean,
-  rightButton: boolean,
-  stopPropagation: Function,
-  preventDefault: Function,
+  delta?: number,
```

Comment:
Why are some of the fields deleted? They are accessed in map-controller.js:
- deltaX
- deltaY
- scale
- rotation

Reply:
Hmm they aren't documented as existing so I wasn't sure if they existed, but given we use them then they certainly do exist. So restored them (but made them optional, because they only exist for some events)